core/src/main/scala/org/apache/spark/api/r/RUtils.scala

```diff
@@ -34,11 +38,15 @@ private[spark] object RUtils {
   }
 
   /**
-   * Get the SparkR package path in various deployment modes.
+   * Get the list of paths for R packages in various deployment modes, of which the first
+   * path is for the SparkR package itself and the second if exists is for R packages
```

Comment:
`path is for the SparkR package itself. The second value is for R packages built as part of Spark Packages, if any exist.  Spark Packages can be provided through the --packages or --jars command line options` or something like that. Very long sentences are hard to read :)